### PR TITLE
CUDA_VISIBLE_DEVICES= (empty) will avoid cuda.is_available check

### PR DESCRIPTION
**Pull request by le-horizon**
disable cuda.is_available check with CUDA_VISIBLE_DEVICES= (empty)
This is useful during hardware issues when the check hangs forever.

### CODE REVIEW

File: alf/utils/common.py [hunk 1]
```diff
@@ -48,6 +48,13 @@
 from . import dist_utils, gin_utils
 
 
+def cuda_is_available():
+    # When CUDA_VISIBLE_DEVICES= (empty), simply skip the check.
+    # This is useful during hardware issues when the check hangs forever.
+    cuda_dev = os.getenv("CUDA_VISIBLE_DEVICES", default=None)
+    return (cuda_dev is None or cuda_dev != "") and torch.cuda.is_available()
```
Comment (emailweixu) [inline alf/utils/common.py:55]: If CUDA_VISIBLE_DEVICES=="", torch.cuda.is_availble() will return False.

Comment (le-horizon) [reply alf/utils/common.py:55]: When hardware works properly then yes, but if there is a hardware or driver problem, then this call could hang.  That's the reason for this change.

File: alf/bin/train_play_test.py [hunk 2]
```diff
@@ -397,349 +397,349 @@ def _test_performance(self, root_dir, test_func):
...
+    #     self._test(
+    #         conf_file='oac_halfcheetah_conf.py',
+    #         skip_checker=self._skip_if_mujoco_unavailable,
+    #         extra_train_params=OFF_POLICY_TRAIN_PARAMS)
+
+    # def test_oac_humanoid(self):
+    #     self._test(
+    #         conf_file='oac_humanoid_conf.py',
+    #         skip_checker=self._skip_if_mujoco_unavailable,
+    #         extra_train_params=OFF_POLICY_TRAIN_PARAMS)
```
Comment (emailweixu) [inline alf/bin/train_play_test.py]: need to uncomment

**Comment by emailweixu** [comment]
> disable cuda.is_available check with CUDA_VISIBLE_DEVICES= (empty)

The PR title is not correct.

File: alf/bin/train_play_test.py (hunk 2 above)
Comment (le-horizon) [reply alf/bin/train_play_test.py]: Done.

**Comment by emailweixu** [comment]
If this is only useful when the driver or hardware is broken, I'd suggest not to merge it. If people has a GPU machine but the driver or hardware is broken, they should fix those instead.

**Pull request closed without merging**